### PR TITLE
update(Row): Reduce padding for inline and compact.

### PR DESCRIPTION
**Pull request by schillerk**
Discussed in Lunar Project Planning (see agenda).

With compact and inline both enabled, before/after padding down to 0.5 * unit.
With inline enabled, no vertical padding.

to: @milesj @stefhatcher @hayes @alecklandgraf

## Screenshots
Screen Shot 2020-02-03 at 1 17 09 PM

## Checklist

- My code follows the style guide of this project.
- I have updated or added documentation accordingly.
- I have read the CONTRIBUTING document.

### CODE REVIEW

**Comment by schillerk** [comment]
Abandoned for https://github.com/airbnb/lunar/pull/297

File: packages/core/src/components/Row/story.tsx [hunk 1]
```diff
@@ -142,6 +142,16 @@ export function allPaddingOptions() {
...
         <Text>A row with no padding.</Text>
       </Row>
+
+      <Row inline after={<Button>Take an action</Button>}>
+        <Text>An inline row with no vertical padding.</Text>
+      </Row>
+
+      <Row compact inline after={<Button>Take an action</Button>}>
+        <Text>
+          A inline and compact row with no vertical padding and low horizontal padding (4px).
```
Comment (stefhatcher) [inline packages/core/src/components/Row/story.tsx]: small nit: `A` -> `An`

fix that before merging please

Comment (schillerk) [reply packages/core/src/components/Row/story.tsx]: Got it, thanks!

**Comment by stefhatcher** [comment]
thank you!

**Comment by milesj** [comment]
@schillerk Can you rebase? I fixed happo running OOM.

**Pull request merged**